src/pages/components/multi-thumb-range.mdx

```diff
@@ -0,0 +1,112 @@
+  A Multi-Thumb Range is a slider with two or more thumbs that each set a value in
+  a group of related values.
+package: '@zendeskgarden/react-forms'
+components:
+  - forms/src/elements/common/Field.tsx
+  - forms/src/elements/common/Hint.tsx
+  - forms/src/elements/common/Label.tsx
+  - forms/src/elements/common/Message.tsx
+  - forms/src/elements/MultiThumbRange.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To choose a range of numbers between minimum and maximum values</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To choose a single number between a range of values, use a [Range](/components/range)
+          instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Multi-Thumb Range component.
+
+import Default from '../../examples/forms/multithumbrange/MultiThumbRangeDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/forms/multithumbrange/MultiThumbRangeDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Disabled
+
+A disabled Multi-Thumb Range prevents user interaction. It doesn’t appear in the tab order,
+can’t receive focus, and may not be read aloud by a screenreader.
+
+import Disabled from '../../examples/forms/multithumbrange/MultiThumbRangeDisabled.tsx';
+import DisabledCode from '!!raw-loader!../../examples/forms/multithumbrange/MultiThumbRangeDisabled.tsx';
+
+<CodeExample code={DisabledCode}>
+  <Disabled />
+</CodeExample>
+
+### Hint text
+
+To provide additional context, use hint text with the label.
+
+import Hint from '../../examples/forms/multithumbrange/MultiThumbRangeHint.tsx';
+import HintCode from '!!raw-loader!../../examples/forms/multithumbrange/MultiThumbRangeHint.tsx';
+
+<CodeExample code={HintCode}>
+  <Hint />
+</CodeExample>
+
+### Validation
+
+Show success, warning, and danger validation messages with the Message component.
+
+import Validation from '../../examples/forms/multithumbrange/MultiThumbRangeValidation.tsx';
+import ValidationCode from '!!raw-loader!../../examples/forms/multithumbrange/MultiThumbRangeValidation.tsx';
+
+<CodeExample code={ValidationCode}>
+  <Validation />
+</CodeExample>
+
+## Configuration
+
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
+
+## API
+
+### Field
+
+<PropSheet components={props.data.mdx.components} componentName="field" />
+
+### Hint
+
+<PropSheet components={props.data.mdx.components} componentName="hint" />
+
+### Label
+
+<PropSheet components={props.data.mdx.components} componentName="label" />
+
+### Message
+
+<PropSheet components={props.data.mdx.components} componentName="message" />
+
+### MultiThumbRange
+
+<PropSheet components={props.data.mdx.components} componentName="multithumbrange" />
+
+import { graphql } from 'gatsby';
+
+export const pageQuery = graphql`
+  query($fileAbsolutePath: String) {
+    ...SidebarPageFragment
```

Comment:
fwiw, a quick glance at the TOC looks like if we bring the `padding-right` down from 20px to 16px, we'll avoid the ellipsis truncation on `MultiThumbRange`. Although, I could be wrong and the overall padding adjustment may not be desirable.